### PR TITLE
Add missing Eth-Consensus-Version headers

### PR DESCRIPTION
**Pull request by saolyn**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

Adds the missing `Eth-Consensus-Version` headers to `GetAggregateAttestationV2` and `ListAttestationsV2`.

**Which issues(s) does this PR fix?**

N/A

**Other notes for review**

**Acknowledgements**

- [x] I have read [CONTRIBUTING.md](https://github.com/prysmaticlabs/prysm/blob/develop/CONTRIBUTING.md).
- [x] I have made an appropriate entry to [CHANGELOG.md](https://github.com/prysmaticlabs/prysm/blob/develop/CHANGELOG.md).
- [x] I have added a description to this PR with sufficient context for reviewers to understand this PR.

### CODE REVIEW

File: beacon-chain/rpc/eth/validator/handlers.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ func (s *Server) GetAggregateAttestationV2(w http.ResponseWriter, r *http.Reques
 		}
 		resp.Data = data
 	}
+	w.Header().Set(api.VersionHeader, version.String(agg.Version()))
```
Comment (rkapka) [inline beacon-chain/rpc/eth/validator/handlers.go:126]: Per https://github.com/ethereum/beacon-APIs/pull/470/files, the version should be `The active consensus version`, not just Phase0/Electra

**Review by rkapka — CHANGES_REQUESTED** [review]

**Comment by james-prysm** [comment]
we also need to add it on [/eth/v2/beacon/pool/attester_slashings](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getPoolAttesterSlashingsV2)

File: beacon-chain/rpc/eth/validator/handlers.go (hunk 1 above)
Comment (saolyn) [reply beacon-chain/rpc/eth/validator/handlers.go:126]: You're right, I'll fix that

**Comment by saolyn** [comment]
> we also need to add it on [/eth/v2/beacon/pool/attester_slashings](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Beacon/getPoolAttesterSlashingsV2)

I checked `getPoolAttesterSlashingsV2` and it is already there.

**Review by james-prysm — APPROVED** [review]

**Review by rkapka — APPROVED** [review]

**Pull request merged**